frontend/index.html

```diff
@@ -142,8 +109,8 @@
 			src="//js.hs-scripts.com/20473940.js"
 		></script>
 		<!-- End of HubSpot Embed Code -->
-
-		<script type="module" src="./src/index.tsx"></script>
+		<script type="module" src="./src/canny.js" defer></script>
```

Comment:
Moved this from body to up here and switched from async to defer to make sure this is executed before the index bundle. Otherwise if index ever gets executed first, references to Canny will fail (unlikely because index is so much bigger, but still possible when cached).